showcase/src/assets/translations/nl.json

```diff
@@ -330,6 +330,7 @@
         },
         "REGULAR": "Normale tabel",
         "WITH_SELECTION": "Tabel met selectie",
+        "WITH_ITEMS_PER_PAGE_SELECTOR": "Tabel met aangepaste items per page",
```

Comment:
I suggest to change this for:
"Tabel met de selector van items per pagina om weer te geven"